extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js

```diff
@@ -182,6 +189,9 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
     /** @private {number|null} */
     this.idOfAdShowing_ = null;
 
+    /** @private {boolean} */
+    this.firstAdShown_ = false;
```

Comment:
let's be consistent about terminology here, i think this mean `viewed`?

Reply:
Done.